services/cacitesting.env

```diff
@@ -1,5 +1,5 @@
 # Core Environment Variables
-export SUBSCRIPTION=85c61f94-8912-4e82-900e-6ab44de9bdf8
+export SUBSCRIPTION=7ca35580-fc67-469c-91a7-68b38569ca6e
```

Comment:
Shouldn't this be a secret?

Reply:
Why? It's just an ID you can't do anything with it without sufficient role based access

Reply:
OK fair